src/Symfony/Bundle/TwigBundle/composer.json

```diff
@@ -24,6 +24,7 @@
         "twig/twig": "~1.28|~2.0"
     },
     "require-dev": {
+        "symfony/console": "~3.3",
```

Comment:
Only to run tests with the last version of FrameworkBundle (https://travis-ci.org/symfony/symfony/jobs/191309256). Another option is to make FrameworkBundle requiring `console`.